features/bootstrap/MinkContext.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace PantheonSystems\PantheonWordPressUpstreamTests\Behat;
+
+use Behat\Behat\Context\TranslatableContext;
+use Behat\Gherkin\Node\TableNode;
+use Behat\MinkExtension\Context\MinkContext as BehatMinkContext;
+
+
+class MinkContext extends BehatMinkContext {
+
+  /**
+   * Build URL, based on provided path.
+   *
+   * @param string $path Relative or absolute URL.
+   * @return string
+   */
+  public function locatePath($path)
+  {
+    if (strtolower(substr($path, 0, '4')) === 'http') {
+      return $path;
+    }
+    $ext = pathinfo($path, PATHINFO_EXTENSION);
+    $url = $this->getMinkParameter('base_url');
+    if (!empty(getenv('RELOCATED_WP_ADMIN')) && (strpos($path, 'wp-admin') !== false || in_array($ext, ['htm', 'html', 'md', 'php', 'txt'], true))) {
```

Comment:
Because I don't have a clean mechanism for getting it. This code runs in CircleCI while WordPress is running on Pantheon.

WordHat, as a Behat extension, can get it's own configuration, including site_url: https://github.com/paulgibbs/behat-wordpress-extension/blob/master/src/Context/RawWordpressContext.php#L52

Here we don't have an extension, just a Context overriding normal Mink. So we don't have separate `base_url` and `site_url` parameters passed in. Perhaps we're better off just taking the time now to replace this class with WordHat rather than documenting how this class is used and the hacky RELOCATED_WP_ADMIN variable.

I had difficulty running Wordhat against Pantheon with the wp-cli driver (Because I couldn't get wp-cli aliases working with Pantheon). But as I look at the documentation again, the Blackbox driver should work fine for what we need (login logic) https://wordhat.info/settings/index.html

Let's talk about this tomorrow doing our working time in the afternoon.